cumulusci/tasks/push/pushfails.py

```diff
@@ -0,0 +1,69 @@
+""" simple task(s) for reporting on push upgrade jobs.
+
+this doesn't use the nearby push_api module, and was just a quick ccistyle
+get the job done kinda moment.
+"""
+
+import re
+import csv
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class PushFailTask(BaseSalesforceApiTask):
+    """ Produce a report of the failed and otherwise anomalous push jobs from
+    a specified push request id, and write it out to a results CSV.  """
```

Comment:
PEP 257 (how do I know that number without looking it up? scary) recommends starting multi-line docstrings with a one-line summary followed by a blank line and then any details. So here that would look like:

```
"""Produce a report of unsuccessful push jobs.

Takes a push request id and writes results to a CSV file.
"""
```

I don't think we need to be real strict about this, but wanted to make sure you knew the Pythonic convention.